src/vs/platform/opener/common/opener.ts

```diff
@@ -32,5 +32,7 @@ export interface IOpenerService {
 export const NullOpenerService: IOpenerService = Object.freeze({
 	_serviceBrand: undefined,
 	registerOpener() { return { dispose() { } }; },
-	open() { return Promise.resolve(false); }
+	open() {
+		return Promise.resolve(false);
+	}
```

Comment:
unnecessary change

Reply:
I'm sorry i was debugging a bit, I have removed the unnecessary formatting!